### PR TITLE
Module1/address

### PR DESCRIPTION
**Pull request by Marin-Andres**

### CODE REVIEW

File: App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx [hunk 1]
```diff
@@ -2,17 +2,197 @@
...
+        });
     }
 
+    saveData() {
+        try {
+            const valid = this.schema.validateSync(this.state.newData);
+            this.props.updateProfileData(this.props.componentId, this.state.newData);
+            this.closeEdit();
+          } catch (error) {
+            console.log("error",error);
```
Comment (kestermvp) [inline App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx]: Please remove this one

**Review by kestermvp — COMMENTED** [review]
Aside from my code feedback, can you please also check why after I updated my details and when I reload the page, the address data will display empty. But after I updated it and reload the page. the content will then show.

**Review by kestermvp — CHANGES_REQUESTED** [review]
Please check my notes and feedback

**Comment by Marin-Andres** [comment]
I'm now refreshing the profile data after an update, I think this is now fixed

**Pull request merged**